app/config/config_prod.yml

```diff
@@ -9,3 +9,29 @@ parameters:
   chameleon_system_core.cache.memcache_use_fallback: false
   chameleon_system_core.cache.include_cache_delete_trace_info: false
   chameleon_system_core.debug.show_view_source_html_hints: false
+
+monolog:
+  handlers:
+    main:
+      type: fingers_crossed
```

Comment:
Does this still fit the upgrade doc with "fingers_crossed is not used anymore"?

Reply:
Removed the note in the upgrade doc.